client/my-sites/site-settings/related-posts/related-content-preview.jsx

```diff
@@ -1,14 +1,21 @@
 import { localize } from 'i18n-calypso';
 import FormLabel from 'calypso/components/forms/form-label';
 
-const RelatedContentPreview = ( { showHeadline, showThumbnails, translate } ) => {
+const RelatedContentPreview = ( {
+	showContext,
+	showDate,
+	showHeadline,
+	showThumbnails,
+	translate,
+} ) => {
 	const posts = [
 		{
 			image: '/calypso/images/related-posts/cat-blog.png',
 			title: translate( 'Big iPhone/iPad Update Now Available', {
 				textOnly: true,
 				context: 'Demo content for related posts',
 			} ),
+			date: 'August 8, 2005',
```

Comment:
I know this may sound really picky but since we are not translating the strings here, can we always display for example yesterday's date, or at least 1st day of the current year, i.e. `January 1, ${new Date().getFullYear()}`? Having `'August 8, 2005'` may create a little bit of an impression that our product is old/out-of-date. 😅

Reply:
Great point! I was thinking about it a bit before, but then decided to keep it the same as we have in WP Admin.

I will merge this PR and then create a new one that will address the date. To make sure this one is already out for Monday CfT start. 🙂

Reply:
> I will merge this PR and then create a new one that will address the date.

PR is ready for review: https://github.com/Automattic/wp-calypso/pull/72996.